ci/get-engine.ps1

```diff
@@ -1,3 +1,9 @@
+param(
+  [string] $unreal_path = "$($gdk_home)\UnrealEngine",
```

Comment:
nit: Could you add a comment up here adding that this is a symlinked directory into a specific engine version within the $engine_cache_directory?

Reply:
Done.